src/libraries/Microsoft.Bcl.TimeProvider/src/System/Threading/Tasks/TimeProviderTaskExtensions.cs

```diff
@@ -0,0 +1,190 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Threading.Tasks
+{
+    public static class TimeProviderTaskExtensions
```

Comment:
Nit: this should have triple-slash documentation and we should probably turn on project-source-of-truth docs for this package.  cc @carlossanlop

Reply:
I'll address that. Thanks for the catch.

Reply:
> we should probably turn on project-source-of-truth docs for this package

FYI, we have an issue with when enable this on partial facade assembly we get some error. I confirmed with @carlosalberto it is known issue and tracked to be fixed at some point.

```Xml
 <UseIntellisensePackageDocXmlFile>false</UseIntellisensePackageDocXmlFile>
```

Reply:
Wrong Carlos 😄

Reply:
Oops! :grinning: